lib/screens/device_monitor.ex

```diff
@@ -92,19 +84,13 @@ defmodule Screens.DeviceMonitor do
     # run at a consistent time past the top of the minute
     send_after =
       now
-      |> truncate_to_interval(1)
+      |> truncate_seconds()
       |> DateTime.add(1, :minute)
       |> DateTime.add(1, :second)
       |> DateTime.diff(now, :millisecond)
 
     Process.send_after(self(), :run, send_after)
   end
 
-  # Truncate a datetime to the given interval size in minutes. For example, when the size is 5,
```

Comment:
Does this rollback some of the interval stuff that was just added?

Reply:
Sort of, yeah 😅 That extra complexity was needed because we were using an interval longer than 1 minute, so just truncating off the seconds wasn't sufficient. Now that checking every minute is fine, it can be simplified back to whence it came.